py/desispec/test/test_photo.py

```diff
@@ -0,0 +1,91 @@
+"""
+tests desispec.io.photo
+
+"""
+import os
+import unittest
+
+import numpy as np
+from desispec.io.photo import gather_targetphot, gather_tractorphot, gather_targetdirs
+
+if 'NERSC_HOST' in os.environ and \
+        os.getenv('DESI_SPECTRO_DATA') == '/global/cfs/cdirs/desi/spectro/data':
```

Comment:
I'm ok with hardcoding /global/cfs/cdirs here for convenience and we can update the NERSC-specific unit tests later if needed.

Reply:
OK. For the record, I coopted this syntax from the `test_fibermap` unit tests.